sdk/test-utils/perfstress/src/options.ts

```diff
@@ -0,0 +1,147 @@
+ */
+export type PerfStressOptionValue = string | number | boolean | undefined;
+
+/**
+ * The structure of a PerfStress option. They represent command line parameters.
+ */
+export interface PerfStressOption {
+  /**
+   * Whether the option is required or not.
+   */
+  required?: boolean;
+  /**
+   * The shortName represents a command line argument that is usually short,
+   * and when sent through the command line, it should have a single dash `-` before the shortName specified.
+   * The value of shortName should not contain dashes.
+   */
+  shortName?: string;
+  /**
+   * The longName represents a command line argument that is usually longer than the shortName,
+   * and when sent through the command line, it should have two dashes `--` before the longName specified.
+   * Options don't need to define longName, since it will be derived from the properties PerfStressOptionDictionary.
+   */
+  longName?: string;
+  /**
+   * The default value that is going to be assigned to the option.
+   */
+  defaultValue?: PerfStressOptionValue;
+  /**
+   * The value specified by the user from the command line after either the shortName or the longName.
+   * If no value was specified, the defaultValue will be used.
+   * If the shortName or longName was specified, but no value was provided, "true" will be set.
+   */
+  value?: PerfStressOptionValue;
+  /**
+   * The description of each option. Descriptions of the assigned options will be shown at the beginning of the test call.
+   * Descriptions of all the available options will be shown if the user sends either --help or -h.
+   */
+  description: string;
+}
+
+/**
+ * A group of options is called PerfStressOptionDictionary,
+ * and is shaped as a plain object to make it easier to access them.
+ *
+ * TNames defines the names of the options. This is necessary to allow TypeScript to suggest the appropriate names
+ * for the options.
+ */
+export type PerfStressOptionDictionary<TNames extends string> = {
+  [longName in TNames]: PerfStressOption;
+};
+
+/**
+ * These are the default options longNames.
+ */
+export type DefaultPerfStressOptionNames =
+  | "help"
+  | "no-cleanups"
+  | "parallel"
+  | "duration"
+  | "warmup"
+  | "iterations"
+  | "no-cleanup"
+  | "milliseconds-to-log"
+  | "sync";
+
+/**
+ * These are the default options in full.
+ */
+export const defaultPerfStressOptions: PerfStressOptionDictionary<DefaultPerfStressOptionNames> = {
+  help: {
+    description: "Shows all of the available options",
+    shortName: "h"
+  },
+  "no-cleanups": {
+    description: "Disables all cleanups"
+  },
+  parallel: {
+    description: "How many of the same test to call at the same time",
+    shortName: "p",
+    defaultValue: 1
+  },
+  duration: {
+    description: "When to stop calling tests at all",
+    shortName: "d",
+    defaultValue: 10
+  },
+  warmup: {
+    description: "Duration of warmup in seconds",
+    shortName: "w",
+    defaultValue: 0
```

Comment:
Please default warmup to 5 seconds.  While the JS engine itself may not require warmup, real-world tests often will (e.g. creating the initial HTTP connection).